assets/ump.json

```diff
@@ -3,8 +3,8 @@
   "description": "This Defold extension allows you to integrate Google's User Messaging Platform (UMP) into your Defold game to manage user consent for ads on Android. It helps you request consent information, show the consent form, and check the user's consent status",
   "forum_url": "https://forum.defold.com/t/defold-google-user-messaging-platform-ump-extension/78722",
   "images": {
-    "hero": "",
-    "thumb": ""
+    "hero": "https://raw.githubusercontent.com/tocaRepo/defold-androidextension-ump/refs/heads/master/ump-hero.png",
```

Comment:
Please submit the images as local files in this repo instead. We want to avoid downloads from external sources.